runtime/calamari/src/fee.rs

```diff
@@ -31,30 +26,6 @@ pub const FEES_PERCENTAGE_TO_TREASURY: u8 = 45;
 pub const TIPS_PERCENTAGE_TO_AUTHOR: u8 = 100;
 pub const TIPS_PERCENTAGE_TO_TREASURY: u8 = 0;
 
-/// Handles converting a weight scalar to a fee value, based on the scale and granularity of the
-/// node's balance type.
-///
-/// This should typically create a mapping between the following ranges:
-///   - [0, MAXIMUM_BLOCK_WEIGHT]
-///   - [Balance::min, Balance::max]
-///
-/// Yet, it can be used for any other sort of change to weight-fee. Some examples being:
-///   - Setting it to `0` will essentially disable the weight fee.
-///   - Setting it to `1` will cause the literal `#[weight = x]` values to be charged.
-pub struct WeightToFee;
-impl WeightToFeePolynomial for WeightToFee {
-    type Balance = Balance;
-    fn polynomial() -> WeightToFeeCoefficients<Self::Balance> {
-        // Refer to the congested_chain_simulation() test for how to come up with the coefficient.
-        smallvec![WeightToFeeCoefficient {
-            coeff_integer: 5000u32.into(),
-            coeff_frac: Perbill::zero(),
-            negative: false,
-            degree: 1,
-        }]
-    }
-}
```

Comment:
This basically expands to constant mutliplier so just using the ConstantMultiplier type